packages/amplify_analytics_web/.flutter-plugins

```diff
@@ -0,0 +1,4 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
Probably not a huge deal for the POC, but ideally the generated files are not committed.